pyatv/protocols/companion/api.py

```diff
@@ -350,3 +417,15 @@ async def fetch_attention_state(self) -> SystemStatus:
             raise exceptions.ProtocolError("missing content")
 
         return SystemStatus(content["state"])
+
+    async def _touch_start(self) -> Mapping[str, Any]:
+        """ Subscribe to touch gestures """
+        self._base_timestamp = time.time_ns()
+        response = await self._send_command("_touchStart",
+            {"_height": TOUCHPAD_HEIGHT, "_tFl": 0, "_width": TOUCHPAD_WIDTH})
+        await asyncio.gather(*self.dispatch("_touchStart", response.get("_c", {})))
```

Comment:
I don't see why you need to use `asyncio.gather` here?

Reply:
correct not needed